specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/StaticSites.json

```diff
@@ -939,38 +947,54 @@
           }
         },
         "x-ms-examples": {
-          "Get app settings of a static site build": {
-            "$ref": "./examples/ListStaticSiteBuildAppSettings.json"
+          "Get overview of database connections for the static site build.": {
+            "$ref": "./examples/GetStaticSiteBuildDatabaseConnection.json"
           }
         }
-      }
-    },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/staticSites/{name}/builds/{environmentName}/listFunctionAppSettings": {
-      "post": {
+      },
+      "put": {
         "tags": [
           "StaticSites"
         ],
-        "summary": "Gets the application settings of a static site build.",
-        "description": "Description for Gets the application settings of a static site build.",
-        "operationId": "StaticSites_ListStaticSiteBuildFunctionAppSettings",
+        "summary": "Create or update a database connection for a static site build",
+        "description": "Description for Create or update a database connection for a static site build",
+        "operationId": "StaticSites_CreateOrUpdateBuildDatabaseConnection",
         "parameters": [
           {
             "$ref": "#/parameters/resourceGroupNameParameter"
           },
           {
             "name": "name",
             "in": "path",
-            "description": "Name of the static site.",
+            "description": "Name of the static site",
             "required": true,
             "type": "string"
           },
           {
             "name": "environmentName",
             "in": "path",
-            "description": "The stage site identifier.",
+            "description": "The stage site identifier",
+            "required": true,
+            "pattern": "^[a-zA-Z0-9]+$",
+            "type": "string"
+          },
+          {
+            "name": "databaseConnectionName",
+            "in": "path",
+            "description": "Name of the database connection",
             "required": true,
+            "pattern": "^[a-zA-Z0-9]+$",
             "type": "string"
           },
+          {
+            "name": "databaseConnectionRequestEnvelope",
+            "in": "body",
+            "description": "A JSON representation of the database connection request properties",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/StaticSiteDatabaseConnectionRequestPropertiesARMResource"
```

Comment:
>StaticSiteDatabaseConnectionRequestPropertiesARMResource [](http://example.com/codeflow?start=37&length=56)

Possibly too late now.... (breaking change?) 
But I'd just hope to give it a name that's simpler like
'DatabaseConnectionRequest'